<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Chose the shape</string>
    </property>
   </widget>
   <widget class="QLabel" name="picture">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>450</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CALCULATE</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>173</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="rb_square">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Square</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_rectangle">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Rectangle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_parallelogram">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Parallelogram</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_rhombus">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Rhombus</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_triangle">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Triangle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_trapezoid">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Trapezoid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_circle">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Circle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_sector">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Sector</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>151</width>
      <height>136</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="entry_a">
       <item>
        <widget class="QLabel" name="label_a">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>a: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_a"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="entry_b">
       <item>
        <widget class="QLabel" name="label_b">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>b: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_b"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="entry_h">
       <item>
        <widget class="QLabel" name="label_h">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>h: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_h"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="entry_r">
       <item>
        <widget class="QLabel" name="label_r">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>r: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_r"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="entry_alpha">
       <item>
        <widget class="QLabel" name="label_alpha">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Î±: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_alpha"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>490</y>
      <width>151</width>
      <height>55</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>RESULT:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="result">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>151</width>
      <height>46</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>FORMULA:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="formula">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>430</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAuthors"/>
   <addaction name="actionClear"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAuthors">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/pic/img/authors.png</normaloff>:/pic/img/authors.png</iconset>
   </property>
   <property name="text">
    <string>Authors</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/pic/img/clear.png</normaloff>:/pic/img/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/pic/img/exit.png</normaloff>:/pic/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
